### DISCUSSION TITLE
Problema con primera entrega

### DISCUSSION THREAD

**Discussion by marcosmyh** [dds-jv/foro#195] [Consultas generales] [answered]
Buenas noches, tuve un inconveniente con la entrega de macowins: primero cree el tag y después hice el commit. El problema es que apenas creé el tag me llegó un mensaje del classroom que decía "All jobs have failed". Después, borré ese tag (ya que no tenía nada para evaluar) y cree uno nuevo con el commit. ¿Debería hacer algo más? Porque cuando creé el segundo tag no me llegó la notificación del classroom.

**Comment by gastonprieto** [comment, marked as answer]
@marcosmyh se ve bien del lado nuestro
